### PR TITLE
Update free-programming-books-ko.md

### PR DESCRIPTION
**Pull request by egaoneko**

### CODE REVIEW

File: free-programming-books-ko.md [hunk 1]
```diff
@@ -30,7 +31,7 @@
 
 * [AN INTRODUCTION TO PROGRAMMING IN GO 한글 번역](http://www.codingnuri.com/golang-book/index.html)
 * [Go Tour 한글 번역](http://go-tour-kr.appspot.com)
-
+* [가장 빨리 만나는 Go 언어](http://www.pyrasis.com/private/2015/06/01/publish-go-for-the-really-impatient-book)
```
Comment (vhf) [inline free-programming-books-ko.md]: Is this book really free? I tried reading a chapter but got redirected to a website selling the book.

File: free-programming-books-ko.md [hunk 2]
```diff
@@ -12,7 +12,8 @@
 * [R](#r)
 * [Ruby](#ruby)
 * [Swift](#swift)
-
+* [Docker](#docker)
+* [Amazon Web Service](#amazon-web-service)
```
Comment (vhf) [inline free-programming-books-ko.md]: Please put these in alphabetical order.

File: free-programming-books-ko.md [hunk 3]
```diff
@@ -85,3 +86,13 @@
...
+
+
+### Docker
+
+* [가장 빨리 만나는 Docker](http://www.pyrasis.com/private/2014/11/30/publish-docker-for-the-really-impatient-book)
+
+
+### Amazon Web Service
+
+* [아마존 웹 서비스를 다루는 기술](http://www.pyrasis.com/private/2014/09/30/publish-the-art-of-amazon-web-services-book)
```
Comment (vhf) [inline free-programming-books-ko.md]: Same for these two sections: please insert them in alphabetical order.

File: free-programming-books-ko.md (hunk 1 above)
Comment (egaoneko) [inline free-programming-books-ko.md]: Yes. It open to the public about the book. "목차" means table of contents and you can read the contents by clicking under the links.

**Comment by vhf** [comment]
Please open this PR on the master branch, not `vhf:double-check-1769`.

**Comment by egaoneko** [comment]
Close this PR for reopen

**Pull request closed without merging**